src/scripts/main.js

```diff
@@ -1,29 +1,56 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  notification.className = `notification ${type}`;
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  const notificationTitle = document.createElement('h2');
+
+  notificationTitle.className = 'title';
+  notificationTitle.textContent = title;
+  notificationTitle.style.whiteSpace = 'nowrap';
+  notification.appendChild(notificationTitle);
+
+  const notificationDescription = document.createElement('p');
+
+  notificationDescription.textContent = description;
+  notification.appendChild(notificationDescription);
+
+  document.body.appendChild(notification);
+
+  setTimeout(() => {
+    notification.style.display = 'none';
```

Comment:
Instead of setting `notification.style.display = 'none';`, consider removing the notification from the DOM using `notification.remove();` to prevent unnecessary elements from remaining in the DOM.